src/CMakeLists.txt

```diff
@@ -523,11 +526,11 @@ SET(OpenMVG_INCLUDE_DIRS
 
 INCLUDE_DIRECTORIES(${OpenMVG_INCLUDE_DIRS})
 
-find_package(Popsift)
-if(POPSIFT_FOUND)
+find_package(PopSift)
```

Comment:
`find_package(PopSift 1.0.0 CONFIG)`
this will require the current version of popsift with modern cmake thus avoiding any mistakes with previous versions.